flake.nix

```diff
@@ -216,13 +216,17 @@
           default = all-gcc;
         };
         apps = {
+          assigner = {
+            type = "app";
+            program = "${self.packages.${system}.evm-assigner}/bin/assigner/assigner";
```

Comment:
why `.evm-assigner`? executable is part of `zkevm-framework`

Reply:
this command works for me:
```
nix run  #assigner
```

I see, that the target is declared in zkevm-framework/tests/libs/assigner_runner/CMakeLists.txt, but it is deployed to the assigner folder...

Reply:
I have corrected the package name now. I have no idea, why it was working before